### ISSUE TITLE
Bitcode Support

### ISSUE THREAD

**Issue by kdmandawe** [pjebs/Obfuscator-iOS#12]
Hi,

I am using Obfuscator library in my project. However, I cannot create an Archive for my app because of the following error:

<img width="711" alt="screen shot 2017-05-25 at 6 11 35 pm" src="https://cloud.githubusercontent.com/assets/4958800/26446850/6c1cfcb4-4178-11e7-8898-43f0427c962d.png">

Are there updates/workaround to address this issue other than disabling bitcode on the target level? 

Thanks,
Kenneth

**Comment by pjebs** [comment]
https://stackoverflow.com/questions/32640149/disable-bitcode-for-project-and-cocoapods-dependencies-with-xcode7

**Comment by kdmandawe** [comment]
Thanks for the link @pjebs. I've actually seen that one before. However, our client(that uses our library) is requiring bitcode support. Any plans of supporting this in the future?

**Comment by pjebs** [comment]
Yes. Configure your cocoa pods to use bitcode. The link tells you how to turn it off. Just do the opposite.
